src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,48 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
+#include <string>
 
 using namespace std;
 
 int t06_homework() {
-
+    set <string> meme;
+    set <string> memez;
+    int onelove=1;
+    int dodo;
+    int hehe=0;
+    int puka=0;
+    string q;
+    string garu;
+    string golum;
+    bool wasted=false;
+    cin >> dodo;
+    for (int i=(onelove-1);i<dodo;i++){cin >> q;meme.insert(q);for (int jop=0;jop<q.length();jop++){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Same expression on both sides of '-'.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4688993)